websites/M/MyTF1/presence.ts

```diff
@@ -0,0 +1,168 @@
+const presence = new Presence({
+	clientId: "1043951049800695808",
+}),
+strings = presence.getStrings({
+	viewHome: "general.viewHome",
+	viewPage: "general.viewPage",
+	viewProfile: "general.viewProfile",
+	watchingVid: "general.watchingVid",
+	browse: "general.browsing",
+	live: "general.live"
+}),
+browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	
+	const presenceData: PresenceData = {
+		largeImageKey: "mytf1",
+		startTimestamp: browsingTimestamp,
+	},
+	path = window.location.pathname,
+	time = await presence.getSetting<boolean>("time");
+
+	if (!time) {
+		delete presenceData.startTimestamp;
+	}
+
+	const browse = (await strings).browse,
+	viewHome = (await strings).viewHome,
+	viewProfile = (await strings).viewProfile,
+	watchingVid = (await strings).watchingVid,
+	live = (await strings).live; 
+
+	if (path == "/") {
+		presenceData.details = viewHome;
+		presenceData.state = browse;
+		delete presenceData.smallImageKey;
+		delete presenceData.smallImageText;
+		presenceData.largeImageKey = "mytf1";
```

Comment:
Lines like these may be unneeded as the attributes are either the same or do not exist in the object yet.